### DISCUSSION TITLE
求助!! v2.3的底模咋有很多checkpoint没找到呢，是我哪里搞错了吗

### DISCUSSION THREAD

**Discussion by san5167** [fishaudio/Bert-VITS2#295] [Q&A] [answered]
### 报错：
ERROR:models:cond.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\DUR_0.pth
INFO:models:Loaded checkpoint 'D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\DUR_0.pth' (iteration 0)
ERROR:models:enc_p.encoder.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:dec.cond.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:enc_q.enc.cond_layer.weight_v is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:flow.flows.0.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:flow.flows.2.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:flow.flows.4.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
ERROR:models:flow.flows.6.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth

![image](https://github.com/fishaudio/Bert-VITS2/assets/116264162/76522831-b787-42fe-a87b-00329f4d7132)

### 底模下载地址
https://huggingface.co/OedoSoldier/Bert-VITS2-2.3/tree/main

当然这个地址的底模（https://openi.pcl.ac.cn/Stardust_minus/Bert-VITS2/modelmanage/model_filelist_tmpl?name=Bert-VITS2_2.3%E5%BA%95%E6%A8%A1）    
也试过了，不过报的其实异常，难受！！
![image](https://github.com/fishaudio/Bert-VITS2/assets/116264162/1df0ea71-ed6d-483c-b529-f9bae562c723)

**Comment by jiangyuxiaoxiao** [comment]
底膜把这些都裁剪掉了，应该是正常的

**Comment by Stardust-minus** [comment]
> ### 报错：
> ERROR:models:cond.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\DUR_0.pth INFO:models:Loaded checkpoint 'D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\DUR_0.pth' (iteration 0) ERROR:models:enc_p.encoder.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:dec.cond.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:enc_q.enc.cond_layer.weight_v is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:flow.flows.0.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:flow.flows.2.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:flow.flows.4.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth ERROR:models:flow.flows.6.enc.spk_emb_linear.weight is not in the checkpoint in D:\data\code\ai\Bert-vits2-V2.3\data\bailu\models\G_0.pth
> 
> ![image](https://private-user-images.githubusercontent.com/116264162/297803107-76522831-b787-42fe-a87b-00329f4d7132.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2MzIwNTMsIm5iZiI6MTcwNTYzMTc1MywicGF0aCI6Ii8xMTYyNjQxNjIvMjk3ODAzMTA3LTc2NTIyODMxLWI3ODctNDJmZS1hODdiLTAwMzI5ZjRkNzEzMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExOVQwMjM1NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMGQ3ZDZjOWU3YmYwYTc3YmQyNDhjNzMyYTQ0NDZjYWVlNWUwMzI3YmZjOTAwOWY1NjMyMWQ3NDk4ZmY3MTQ5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2-6LnQzvBQsaXHv2lTZf9d7YVym9UEtDDqkGSOJcVK0)
> 
> ### 底模下载地址
> https://huggingface.co/OedoSoldier/Bert-VITS2-2.3/tree/main
> 
> 当然这个地址的底模（[https://openi.pcl.ac.cn/Stardust_minus/Bert-VITS2/modelmanage/model_filelist_tmpl?name=Bert-VITS2_2.3%E5%BA%95%E6%A8%A1）](https://openi.pcl.ac.cn/Stardust_minus/Bert-VITS2/modelmanage/model_filelist_tmpl?name=Bert-VITS2_2.3%E5%BA%95%E6%A8%A1%EF%BC%89) 也试过了，不过报的其实异常，难受！！ ![image](https://private-user-images.githubusercontent.com/116264162/297804454-1df0ea71-ed6d-483c-b529-f9bae562c723.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2MzIwNTMsIm5iZiI6MTcwNTYzMTc1MywicGF0aCI6Ii8xMTYyNjQxNjIvMjk3ODA0NDU0LTFkZjBlYTcxLWVkNmQtNDgzYy1iNTI5LWY5YmFlNTYyYzcyMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExOVQwMjM1NTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xYzI3MTE0ZTljYzUyZGZkYjc1ZDg5OGMwYzA0NTA5MDQ5Njk4OGNkOTdkY2RkYTk2ZDE3MzcxZTYzZTEwNmQ2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.mp0q5jnhPYTTpIguWvMlEHIyKjSiGiPNqIUXRoLGfxk)

gin channel没对上

**Reply by san5167** [reply]
不是很理解，是代码逻辑问题吗。这个问题可以怎么解决呢。

**Reply by Stardust-minus** [reply, marked as answer]
config的gin_channel字段改512即可

**Reply by andymethane** [reply]
您好，我在复现Extra-fix版本时也遇到了类似的问题，请问怎么解决呢？
ERROR:model_fix:enc_p.encoder.spk_emb_linear.weight is not in the checkpoint
ERROR:model_fix:dec.cond.weight is not in the checkpoint
ERROR:model_fix:enc_q.enc.cond_layer.weight_v is not in the checkpoint
ERROR:model_fix:flow.flows.0.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:model_fix:flow.flows.2.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:model_fix:flow.flows.4.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:model_fix:flow.flows.6.enc.spk_emb_linear.weight is not in the checkpoint
ERROR:model_fix:sdp.cond.weight is not in the checkpoint
ERROR:model_fix:dp.cond.weight is not in the checkpoint
ERROR:model_fix:emb_g.weight is not in the checkpoint
我采用的是这个底模https://openi.pcl.ac.cn/Stardust_minus/Bert-VITS2/modelmanage/model_filelist_tmpl?name=Bert-VITS2_%E4%B8%AD%E6%96%87%E7%89%B9%E5%8C%96%E4%BF%AE%E5%A4%8D%E5%BA%95%E6%A8%A1
选择的代码是https://github.com/fishaudio/Bert-VITS2/tree/Extra-Fix

**Comment by Stardust-minus** [comment]
> 不是很理解，是代码逻辑问题吗。这个问题可以怎么解决呢。
